owncloudTestUtil/src/main/java/com/owncloud/android/testutil/OCOAuthParams.kt

```diff
@@ -0,0 +1,6 @@
+package com.owncloud.android.testutil
+
+val AUTH_TOKEN_TYPE = "owncloud.oauth2.access_token"
+val ACCESS_TOKEN = "Asqweh12p93yehd10eu"
+val REFRESH_TOKEN = "P3sd19DSsjdp1jwdd1"
+val SCOPE = "email"
```

Comment:
I would move this to OCAccount since its related with Accounts. 

BTW, adding OC_ as prefix and making them const